docs/rum_getting_started.md

```diff
@@ -0,0 +1,119 @@
+
+## Setup
+
+
+1. Declare [dd-sdk-android][1] as a dependency in your `build.gradle` file:
+
+    ```conf
+    repositories {
+        maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+    }
+
+    dependencies {
+        implementation "com.datadoghq:dd-sdk-android:x.x.x"
+    }
+    ```
+
+2. [Specify application details in Datadog][2] to generate a unique Datadog application ID and client token.
+
+{{< img src="docs/images/screenshot_rum.png" alt="RUM Event hierarchy" style="width:50%;border:none" >}}
+
+To ensure the safety of your data, you must use a client token: you cannot use [Datadog API keys][3] to configure the `dd-sdk-android` library, as they would be exposed client-side in the Android application APK byte code. For more information about setting up a client token, see the [Client Token documentation][4].
+
+3. Initialize the library with application context and start sending data:
+
+    {{< tabs >}}
+    {{% tab "US" %}}
+
+   
+```kotlin
+   class SampleApplication : Application() {
+       override fun onCreate() {
+           super.onCreate()
+
+           val configuration = Configuration.Builder(
+            rumEnabled = true,
+            crashReportsEnabled = true
+        )
+                        .trackInteractions()
+                        .trackLongTasks(durationThresold)
+                        .useViewTrackingStrategy(strategy)
+                        .build()
+          val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>)
+          Datadog.initialize(this, credentials, configuration, trackingConsent)
+
+       }
+   }
+```
+
+    {{% /tab %}}
+    {{% tab "EU" %}}
+```kotlin
+class SampleApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+         val configuration = Configuration.Builder(
+            rumEnabled = true,
+            crashReportsEnabled = true
+        )
+                        .trackInteractions()
+                        .trackLongTasks(durationThresold)
+                        .useViewTrackingStrategy(strategy)
+                        .useEUEndpoints()
+                        .build()
+        val credentials = Credentials(<CLIENT_TOKEN>,<ENV_NAME>,<APP_VARIANT_NAME>,<APPLICATION_ID>)
+        Datadog.initialize(this, credentials, configuration, trackingConsent)
+          
+    }
+}
+```
+    {{% /tab %}}
+    {{< /tabs >}}
+
+Learn more about [`ViewTrackingStrategy`][5] to enable automatic tracking of all your views (activities, fragments ,etc.), [`trackingConsent`][6] to add GDPR compliance for your EU users, and [other configuration options][7] to initialize the library.
+
+Note that in the credentials required for initialization, your application variant name is also required. This is important because it enables the right ProGuard `mapping.txt` file to be automatically uploaded at build time to be able to view de-obfuscated stack traces. [Guide to uploading android source mapping files][8].
```

Comment:
```suggestion
Note that in the credentials required for initialization, your application variant name is also required. This is important because it enables the right ProGuard `mapping.txt` file to be automatically uploaded at build time to be able to view de-obfuscated stack traces. For more information see the [guide to uploading Android source mapping files][8].
```